sunpy/physics/transforms/differential_rotation.py

```diff
@@ -85,9 +81,335 @@ def diff_rot(ddays, latitude, rot_type='howard', frame_time='sidereal'):
 
         #This is in micro-radians / sec
         rotation_rate = A + B * sin2l + C * sin4l
-        rotation_deg = rotation_rate * 1e-6  * delta_seconds / np.deg2rad(1)
+        rotation_deg = rotation_rate * 1e-6 * delta_seconds / np.deg2rad(1)
 
     if frame_time == 'synodic':
         rotation_deg -= 0.9856 * delta_days
 
-    return Longitude((np.round(rotation_deg.value,4)),u.deg)
+    #return Longitude((np.round(rotation_deg, 4)), u.deg)
+    return np.round(rotation_deg, 4) * u.deg
+
+
+@quantity_input(x=u.arcsec, y=u.arcsec)
+def rot_hpc(x, y, tstart, tend, spacecraft=None, frame_time='synodic', rot_type='howard', **kwargs):
+    """Given a location on the Sun referred to using the Helioprojective
+    Cartesian co-ordinate system (typically quoted in the units of arcseconds)
+    use the solar rotation profile to find that location at some later or
+    earlier time.
+
+    Parameters
+    -----------
+    x : `~astropy.units.Quantity`
+        helio-projective x-co-ordinate in arcseconds (can be an array)
+    y : `~astropy.units.Quantity`
+        helio-projective y-co-ordinate in arcseconds (can be an array)
+
+    tstart : `sunpy.time.time`
+        date/time to which x and y are referred.
+    tend : `sunpy.time.time`
+        date/time at which x and y will be rotated to.
+    spacecraft : { None | "soho" | "stereo_a" | "stereo_b" }
```

Comment:
Travis!

https://travis-ci.org/sunpy/sunpy/jobs/49640324

*Configuration file `/etc/rsyslog.conf'*

* ==> Modified (by you or by a script) since installation.*

* ==> Package distributor has shipped an updated version.*

* What would you like to do about it ? Your options are:*

* Y or I : install the package maintainer's version*

* N or O : keep your currently-installed version*

* D : show the differences between the versions*

* Z : start a shell to examine the situation*

* The default action is to keep your current version.*

**** rsyslog.conf (Y/I/N/O/D/Z) [default=N] ? *

*No output has been received in the last 10 minutes, this potentially
indicates a stalled build or something wrong with the build itself.*

*The build has been terminated*


On Thu, Feb 5, 2015 at 9:06 AM, Stuart Mumford <notifications@github.com>
wrote:

> In sunpy/physics/transforms/differential_rotation.py
> <https://github.com/sunpy/sunpy/pull/1281#discussion_r24164477>:
>
> > +    Cartesian co-ordinate system (typically quoted in the units of arcseconds)
> > +    use the solar rotation profile to find that location at some later or
> > +    earlier time.
> > +
> > +    Parameters
> > +    -----------
> > +    x : `~astropy.units.Quantity`
> > +        helio-projective x-co-ordinate in arcseconds (can be an array)
> > +    y : `~astropy.units.Quantity`
> > +        helio-projective y-co-ordinate in arcseconds (can be an array)
> > +
> > +    tstart : `sunpy.time.time`
> > +        date/time to which x and y are referred.
> > +    tend : `sunpy.time.time`
> > +        date/time at which x and y will be rotated to.
> > +    spacecraft : { None | "soho" | "stereo_a" | "stereo_b" }
>
> You should document the API for overwriting the values of B0 and L0 etc,
> as these values can be extracted from STEREO headers?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1281/files#r24164477>.
>